SwiftDemo/SwiftDemo/ViewControllers/MainViewController/ViewController.swift

```diff
@@ -0,0 +1,190 @@
+            switch self {
+            case .SetEnabled:
+                return ("Set Enabled", .Switch)
+            case .TrackEvent:
+                return ("Track Event", .Details)
+            case .TrackEventWithProperties:
+                return ("Track Event with Properties", .Details)
+            case .TrackPage:
+                return ("Track Page", .Details)
+            case .TrackPageWithProperties:
+                return ("Track Page with Properties", .Details)
+            }
+        }
+        
+        static let allCases = [MSAnalyticsCases.SetEnabled, MSAnalyticsCases.TrackEvent, MSAnalyticsCases.TrackEventWithProperties]
+    }
+    
+    enum MSCrashesCases : Int{
+        case SetEnabled, GenerateTestCrash, AppCrashInLastSession
+        var cellSetting : (title:String, type:MSCellType){
+            switch self {
+            case .SetEnabled:
+                return ("Set Enabled", .Switch)
+            case .GenerateTestCrash:
+                return ("Generate Test Crash", .Details)
+            case .AppCrashInLastSession:
+                return ("App Crash in Last Session", .Details)
+            }
+        }
+        
+        static let allCases = [MSCrashesCases.SetEnabled, MSCrashesCases.GenerateTestCrash, MSCrashesCases.AppCrashInLastSession]
+    }
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.tableView.estimatedRowHeight = 50;
+        self.tableView.sectionHeaderHeight = 50;
+        self.tableView.rowHeight = UITableViewAutomaticDimension
+        self.tableView.register(MSTitleTableViewCell.nib(), forCellReuseIdentifier: MSTitleTableViewCell.name())
+        self.tableView.register(MSSwitchTableViewCell.nib(), forCellReuseIdentifier: MSSwitchTableViewCell.name())
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    func showAlertWithMessage(title:String, message:String){
+        let alert = UIAlertController.init(title: title, message: message, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "Okay", style: .cancel, handler: nil))
+        self.present(alert, animated: true, completion: nil)
+    }
+
+}
+
+extension ViewController : UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return section == 0 ? MSAnalyticsCases.allCases.count : MSCrashesCases.allCases.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cellSetting = indexPath.section == 0 ?  MSAnalyticsCases.allCases[indexPath.row].cellSetting : MSCrashesCases.allCases[indexPath.row].cellSetting
+        if cellSetting.type == .Switch{
```

Comment:
Please follow the code style of the existing SDK codebase and add spaces before opening curly braces in `if` statements and before and after `else`